src/main/java/com/algotrading/backtesting/replay/ActiveTradedStocksReader.java

```diff
@@ -0,0 +1,44 @@
+package com.algotrading.backtesting.replay;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ActiveTradedStocksReader {
+
+    private final String classFilePath;
+
+    public ActiveTradedStocksReader(String classFilePath) {
+        this.classFilePath = classFilePath;
+    }
+
+    public Map<String, List<String>> constructYearToTickerListFromFiles() throws IOException {
```

Comment:
This converts the files to a map of year string to the list of ticker strings

Reply:
Yes